scripts/utils/archiveRenv.R

```diff
@@ -0,0 +1,12 @@
+# copies current renv.lock to the archive and appends current timestamp
+local({
+  stopifnot(`No renv active. Try starting the R session in the repo root.` = !is.null(renv::project()))
+
+  # update renv.lock
+  renv::snapshot(prompt = FALSE)
+
+  dir.create(here::here("archive"), showWarnings = FALSE)
+  datetime <- format(Sys.time(), "%Y-%m-%dT%H%M%S")
+  file.copy(renv::paths$lockfile(), here::here("archive", paste0(datetime, "_renv.lock")))
```

Comment:
`here::here` fails in `output/*/magicc` because those directories are special-cursed and contain a `.git` folder. Could that be a problem?

Reply:
I don't think so. This would only cause problems if the working directory was in a magicc folder when here::here is called, but that should never be the case. Did you actually run into this problem or did you just anticipate this problem from reading the here::here documentation?

Reply:
I anticipated the problem because the `here::here` logic seems a little like black magic. It effectively means if you delete the `.git` folder, it stops working, right? So we shouldn't distribute REMIND releases as tarballs in the future, for example. I mean, I like the idea of `here::here`, and maybe if it works, it works, but it seems a bit fickle to me. Maybe we need a `.here` file in the remind main folder, just to be safe?

Reply:
I added calls to [here::i_am](https://here.r-lib.org//reference/i_am.html), now it works even if `.git` is not present, and it will fail with a somewhat helpful error when here::here is called from outside of a remind (sub)folder

Reply:
Also it now even works when called from `output/*/magicc`